knowledge-graph/src/main/scala/io/renku/knowledgegraph/datasets/model.scala

```diff
@@ -49,7 +49,7 @@ object model {
                                       sameAs:           SameAs,
                                       versions:         DatasetVersions,
                                       maybeDescription: Option[Description],
-                                      creators:         Set[DatasetCreator],
+                                      creators:         List[DatasetCreator],
```

Comment:
Is there a reason this isn't a NonEmptyList?

Reply:
It's only due to pragmatic reasons :) If I'd like it to be a NonEmptyList, I'd need to change a bit of the way how the data is collected in the free-text DS search. At the end, this data structure is used only for transferring data between the sparql query and HTTP response, so it's not that bad.